Formula/tldr.rb

```diff
@@ -0,0 +1,20 @@
+class Tldr < Formula
+  desc "Simplified and community-driven man pages"
+  homepage "https://tldr-pages.github.io"
+  url "https://github.com/tldr-pages/tldr-cpp-client/archive/v1.3.0.tar.gz"
+  sha256 "6210ece3f5d8f8e55b404e2f6c84be50bfdde9f0d194a271bce751a3ed6141be"
+  head "https://github.com/tldr-pages/tldr-cpp-client.git"
+
+  depends_on "curl"
```

Comment:
Is it guaranteed to be decently up to date? I remember there were incompatibilities.